modules/understanding-process-id-limits.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-configuring-pid-limits.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="understanding-process-id-limits_{context}"]
+= Understanding process ID limits
+
+A process identifier (PID) is a unique identifier assigned by the Linux kernel to each process or thread currently running on a system. The number of processes that can run simultaneously on a system is limited to 4,194,304 by the Linux kernel, and may additionally be affected by limited access to other system resources such as memory, CPU, and disk space.
+
+In {product-title}, there are two supported limits for PID usage to consider when you schedule work on your cluster:
+
+Maximum number of PIDs per pod (default: 4,096 in {product-title} 4.11 and higher)::
+This is controlled by the `podPidsLimit` parameter set on the node.
```

Comment:
```suggestion
This limit is controlled by the `podPidsLimit` parameter set on the node.
```
- Per the ISG, avoid ambigous pronoun references. It's unclear here what "this" refers to. This what?

cc @shipsing